changelog/_unreleased/2022-09-14-fix_tos-not-mandatory-for-contact.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Checkbox for Contact/Newsletter: agreeing 
+author: jonas-sfx
+author_email: jonas@sfxonline.de
+author_github: jonas-sfx
+---
+# Storefront
+* Changed: `src/Storefront/Resources/snippet/de_DE/storefront.de-DE.json`
```

Comment:
Please update the changelog according to the latest changes.

Reply:
@keulinho changelog is updated now :smiley: